src/workerd/io/actor-sqlite.h

```diff
@@ -20,8 +20,12 @@ class ActorSqlite final: public ActorCacheInterface {
 
 public:
   ActorSqlite(SqliteDatabase::Vfs& vfs, kj::PathPtr path)
-      : db(vfs, path, kj::WriteMode::CREATE | kj::WriteMode::MODIFY | kj::WriteMode::CREATE_PARENT),
-        kv(db) {}
+      : db(kj::heap<SqliteDatabase>(vfs, path,
+            kj::WriteMode::CREATE | kj::WriteMode::MODIFY | kj::WriteMode::CREATE_PARENT)),
+        kv(*db) {}
+  ActorSqlite(kj::Own<SqliteDatabase> dbParam)
```

Comment:
nit: Perhaps we should just have this constructor? If we have a default `WriteMode` for `ActorSqlite`, that's pretty trivial to chuck into a constant.

Also, `explicit` please! No sense tolerating a `nullptr` conversion.

Reply:
Done.